src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx

```diff
@@ -188,6 +184,31 @@ const DefinitionList: React.FC<DLProps> = ({ thread, showDate }) => {
               </div>
             }
           </>
+          : thread.type === "DEBRIEFING" ?
+          <>
+            <div>
+              <dt>Auteur</dt>
+              <dd>{ value.author }</dd>
+            </div>
+            <div>
```

Comment:
No checks?

Reply:
For `undefined`?

Reply:
Yes

Reply:
It's always a string according to the generated types